R/.importConservation.R

```diff
@@ -0,0 +1,45 @@
+.importConservation <- function(fName) {
+    # Validate fName if file and stuff
+
+    # variable names subject to changes
+
+    # Setup
+    open_file(fName)
+    readline(n = 1)
+    currPos <- extract from read line
+    chromosomeNum <- extract from read line
+    # or could specify from function signature
+    outName <- "chromosome + chromosomenum.RDS"
+
+    # Whether to store coordinate with score
+    chromosomeScores <- vector or matrix size 20000
```

Comment:
You need a matrix: one column has chromosome coordinates, the other has conservation values. However if you want to go the way of making a vector of whole chromosome length, you'll need to initialize the whole 100,000,000+ - length vector at the beginning.